### ISSUE TITLE
Controllers and trackers spawning order

### ISSUE THREAD

**Issue by Eirenliel** [SlimeVR/SlimeVR-OpenVR-Driver#6]
Apps like VRChat require controllers to be added before FBT trackers. We need to insure at all times that

* Trackers spawning is slightly delayed after SteamVR starts
* If controllers are handled by SlimeVR, they have to be spawpned first

**Comment by GapVR** [comment]
**Problem**: VRChat world gimmicks/game controls do not work, Vive tracker puck seen on right hand.
**Workaround**: Turn on controllers before starting the SlimeVR Server.

Posting this here since my problem with VRChat is likely related to this issue:

Using Vive 1.0 Wand Controllers with SteamVR on Windows 10.

Had an issue with VRChat not recognizing my controller input for Udon Input Events^ for World gimmicks/game controls/World specific menus. The cause was found to be me turning the controllers on _after_ the SlimeVR Server was started. It also placed a Vive Tracker puck (which is visible in VRC when large menus are opened, but not in the SteamVR void) on my right hand (likely because that is switched on first).

The pair of controllers (I haven't tested multiple pairs yet) turned on _before_ the SlimeVR Server is registered as the only pair of controllers recognized by VRC and any controllers swapped in _after_ exhibits the problem above.

^ [Udon Input Events](https://docs.vrchat.com/docs/input-events) are device specific and independent of SteamVR control bindings.

**Comment by ButterscotchV** [comment]
I don't think this is an issue anymore is it? I haven't seen any mention of it in years.
